lib/tracker.coffee

```diff
@@ -0,0 +1,48 @@
+# contants
+analyticsWriteKey = 'pDV1EgxAbco4gjPXpJzuOeDyYgtkrmmG'
+
+# imports
+_ = require 'underscore-plus'
+
+module.exports =
+class Tracker
+
+  constructor: (@analyticsUserIdConfig) ->
+    {allowUnsafeEval} = require 'loophole'
+    @pkg ?= require("../package.json")
+
+    # Setup Analytics
+    Analytics = null
+    allowUnsafeEval -> Analytics = require 'analytics-node'
```

Comment:
Can we cache `analytics-node` in this function? 
e.g. `Analytics ?= require 'analytics-node'`